tests/phpunit/tests/block-template.php

```diff
@@ -275,6 +275,82 @@ public function test_get_the_block_template_html_does_not_generally_enforce_loop
 		$this->assertSame( array( false, true ), $in_the_loop_logs, 'Main query loop was triggered incorrectly' );
 	}
 
+	/**
+	 * @ticket 58319
+	 *
+	 * @covers ::get_block_theme_folders
+	 * @dataProvider data_get_block_theme_folders
```

Comment:
```suggestion
	 * @ticket 58319
	 *
	 * @covers ::get_block_theme_folders
	 *
	 * @dataProvider data_get_block_theme_folders
	 *
	 * @param string   $theme    The theme's stylesheet.
	 * @param string[] $expected The expected associative array of block theme folders.
	 * @param string   $message The message for when an assertion fails.
```
- Some spacing for the docblock per [this ticket's outline](https://core.trac.wordpress.org/ticket/56070)
- Document test method parameters to make it easier for contributors to know what data is being dealt with at a glance.